### DISCUSSION TITLE
Standalone versions analyzed with virustotal

### DISCUSSION THREAD

**Discussion by ruliworst** [Purfview/whisper-standalone-win#91] [Q&A] [answered]
Hello,

I've analyzed the executables with virustotal and I have these results:
For version Whisper-Faster_r160:

<img width="1462" alt="Screenshot 2023-10-31 at 12 46 14" src="https://github.com/Purfview/whisper-standalone-win/assets/66559228/c4e42d02-71ae-4ea1-91c5-be63c6180658">

For version Whisper-Faster_r153:
<img width="1469" alt="Screenshot 2023-10-31 at 12 50 52" src="https://github.com/Purfview/whisper-standalone-win/assets/66559228/c3776c30-4237-458a-acbc-ec2f02ef0b85">

Why do the executables have those warnings? Are they safe to run?

Thanks in advance.

**Comment by Purfview** [comment, marked as answer]
Link for [r153 file report](https://www.virustotal.com/gui/file/a95278918116af321ca042f7edb1a80b9e1b46e25527bf45872489b81c8ee788). [after reanalysis detections are down to 3]
Link for [r160 file report](https://www.virustotal.com/gui/file/1380efb58d414cc17d35e15b5a16b08e5445e5cfb5a3052a9567df4e28bb7a35). [report differs from r153 because exe was compiled with newer pyinstaller]
Link for [r160.4 file report](https://www.virustotal.com/gui/file/531547166a94b9c5d6cce13b8b910d79be602f51c17af870984d2f40dcfe459b)
Link for [r160.5 file report](https://www.virustotal.com/gui/file/403ce3c8331c7abcd22a82fed90898660ffa792bb65c0bd6899592f66c022ffd)

> Why do the executables have those warnings?

Those warnings indicates crappy [anti]Virus software, that's undocumented feature. 😉


> Are they safe to run?

If it's downloaded from this repo then it's safe.


UPDATE:
If you use one of those "crappy [anti]Virus software" then you should notify the problem to the company producing the erroneous detection, they are the only ones that can fix the issue.

**AVG:** https://www.avg.com/en-gb/false-positive-file-form#pc
**Avast:** https://www.avast.com/en-gb/false-positive-file-form.php#pc
**Zillya:** https://zillya.com/support [for files >50MB -> support@zillya.com ]
**Gridinsoft:** https://support-antimalware.com/open.php
**Jiangmin:** [info1](https://forums.malwarebytes.com/topic/284196-is-this-malware-or-what/#elControls_1504120_menu) and [info2](https://www.autohotkey.com/boards/viewtopic.php?f=17&t=62266).

**BitDefender:** https://www.bitdefender.co.uk/business/submit
**Emsisoft:**  https://www.emsisoft.com/en/help/contact/
**GData:** https://www.gdatasoftware.com/faq/consumer/submit-a-suspicious-file-app-or-url

**Fortinet:** https://www.fortiguard.com/faq/classificationdispute
**DeepInstinct:** vt-fps-requests@deepinstinct.com
**SecureAge:** https://www.secureage.com/support/report-false-positive
**Varist:** support@varist.com and SampleFP@avsubmit.com
**Webroot:** https://www.webroot.com/us/en/business/support/vendor-dispute-contact-us
**Microsoft:** https://www.microsoft.com/en-us/wdsi/filesubmission


https://docs.virustotal.com/docs/false-positive-contacts

**Reply by ruliworst** [reply]
Okey, thank you very much for your fast reply.

**Comment by Purfview** [comment]
![Screenshot_1](https://github.com/SubtitleEdit/subtitleedit/assets/69023953/6e8262bf-3c82-4575-829c-417f8b01fb9e)

With pyinstaller v6.2.0 there were only 1 false positive the last week, now it jumped to 9.
Let's follow progress of this...

Link to [r160.5 file report](https://www.virustotal.com/gui/file/403ce3c8331c7abcd22a82fed90898660ffa792bb65c0bd6899592f66c022ffd).


![Screenshot_5](https://github.com/Purfview/whisper-standalone-win/assets/69023953/8b8de95b-e6a3-4230-9715-f96d503b46bb)

Link to [r160.7 file report](https://www.virustotal.com/gui/file/cf8cff7a45651119a315967a070d35d13556ce10571db460c761b60e104b2135).

**Comment by wegylexy** [comment]
I simply add the exe to my McAfee exclude list, and it is now happy.

**Reply by Purfview** [reply]
What "exe" version do you use?

**Comment by Purfview** [comment]
Some magic, no "viruses" in `r167.2`. Let's see for how long. 😆

![alt text](https://i.imgur.com/zCwRNOH.png)

Link to [r167.2 file report](https://www.virustotal.com/gui/file/b758831bc1cfed31e0b6a4dc636926f28f85c0c46f603e604277aa0ffbdd2bae).
